lib/matplotlib/backends/qt_compat.py

```diff
@@ -193,14 +109,10 @@ def is_pyqt5():
 
 
 # These globals are only defined for backcompatibility purposes.
-ETS = dict(pyqt=(QT_API_PYQTv2, 4), pyside=(QT_API_PYSIDE, 4),
-           pyqt5=(QT_API_PYQT5, 5), pyside2=(QT_API_PYSIDE2, 5))
+ETS = dict(pyqt5=(QT_API_PYQT5, 5), pyside2=(QT_API_PYSIDE2, 5))
```

Comment:
I know that some people who do qt application embedding import qt through Matplotlib (that way your Qt binding and Matplotlib's Qt bindings are always the same ;) ).  Who knows what they are using from this module so even though we no longer use this (ETS is a reference to enthought and their convention of controling which qt bindings to use which I assume we followed so that we would stay in sync with traitui and friends if used together).  

We still follow their env convention (see around L30), but no longer use this particular data structure as part of the mapping.  It is less work to carry around an extra dictionary than it is to deprecate it!